src/backend/distributed/planner/multi_explain.c

```diff
@@ -108,6 +109,11 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	instr_time planDuration;
 	Query *originalQuery = NULL;
 	RelationRestrictionContext *restrictionContext = NULL;
+#if PG_VERSION_NUM >= 90600
+	int cursorOptions = into ? 0 : CURSOR_OPT_PARALLEL_OK;
+#else
+	int cursorOptions = 0;
```

Comment:
I think the reason I kept it as-is was to avoid having to add a line just to declare `localQuery` (otherwise we'd be mixing declarations and code). But I can do that, sure.